packages/@react-stately/color/intl/ar-AE.json

```diff
@@ -8,7 +8,7 @@
   "red": "أحمر",
   "saturation": "التشبع",
   "colorName": "{transparency} {lightness} {chroma} {hue}",
-  "transparency": "{percent} transparent",
+  "transparentColorName": "{lightness} {chroma} {hue}, {percentTransparent} transparent",
   "very dark": "very dark",
```

Comment:
@devongovett English is correct, but the localized still have **{transparency}** in the **colorName**.

Our automation will correct that when updating translations, just mentioning in case issue before then,